server/src/test/java/umm3601/UserFunctionalSpec.java

```diff
@@ -16,7 +16,7 @@
 import io.javalin.http.HttpCode;
 import kong.unirest.HttpResponse;
 import kong.unirest.Unirest;
-import umm3601.Server;
+import umm3601.user.UserDatabase;
```

Comment:
We need both these imports, I think.

```suggestion
import umm3601.Server;
import umm3601.user.UserDatabase;
```

Reply:
We don't need the `Server` import because we're now in the `umm3601` where `Server` lives. Everything in you package is automatically imported, and you only need to import things in other packages.

Reply:
Oh, maybe we don't if we're in the same package. Never mind, ignore me! 😅